Tasks/RunJMeterLoadTest/CltTasksUtility.ps1

```diff
@@ -1,338 +1,350 @@
-			"maxVusers":$vu,
-			"loadGenerationGeoLocations":[
-				{"Location":"$geoLocation","Percentage":100}
-			]
-		}
-	}
+    $tdjson = @"
+    {
+        "dropType": "InplaceDrop",
+        "loadTestDefinition":{
+            "loadTestName":"$name",
+            "runDuration":$duration,
+            "urls":["$homepage"],
+            "browserMixs":[
+                {"browserName":"Internet Explorer 11.0","browserPercentage":60.0},
+                {"browserName":"Chrome 2","browserPercentage":40.0}
+            ],
+            "loadPatternName":"Constant",
+            "maxVusers":$vu,
+            "loadGenerationGeoLocations":[
+                {"Location":"$geoLocation","Percentage":100}
+            ]
+        }
+    }
 "@
 
-	return $tdjson
+    return $tdjson
 }
 
 function CreateTestDrop($headers, $CltAccountUrl)
 {
-	$uri = [String]::Format("{0}/_apis/clt/testdrops?{1}", $CltAccountUrl, $global:apiVersion)
-	$drop = InvokeRestMethod -contentType "application/json" -uri $uri -headers $headers -method Post -body "{ ""dropType"": ""TestServiceBlobDrop"" }"
-	return $drop
+    $uri = [String]::Format("{0}/_apis/clt/testdrops?{1}", $CltAccountUrl, $global:apiVersion)
+    $drop = InvokeRestMethod -contentType "application/json" -uri $uri -headers $headers -method Post -body "{ ""dropType"": ""TestServiceBlobDrop"" }"
+    return $drop
 }
 
 function GetTestDrop($headers, $drop, $CltAccountUrl)
 {
-	$uri = [String]::Format("{0}/_apis/clt/testdrops/{1}?{2}", $CltAccountUrl, $drop.id, $global:apiVersion)
-	$testdrop = InvokeRestMethod -contentType "application/json" -uri $uri -headers $headers
+    $uri = [String]::Format("{0}/_apis/clt/testdrops/{1}?{2}", $CltAccountUrl, $drop.id, $global:apiVersion)
+    $testdrop = InvokeRestMethod -contentType "application/json" -uri $uri -headers $headers
 
-	return $testdrop
+    return $testdrop
 }
 
 function UploadTestDrop($testdrop, $src)
 {   
-	$dest = $testdrop.accessData.dropContainerUrl
-	$sas = $testdrop.accessData.sasKey
-
-	$azcopy = Get-ToolPath -Name "AzCopy\AzCopy.exe"
-	Write-Verbose "Calling AzCopy = $azcopy" -Verbose
-
-	$azlog = ("{0}\..\azlog" -f $src)
-	$args = ("/Source:`"{0}`" /Dest:{1} /DestSAS:{2} /S /Z:`"{3}`"" -f $src, $dest, $sas, $azlog)
-
-	Invoke-Tool -Path $azcopy -Arguments $args
+    $dest = $testdrop.accessData.dropContainerUrl
+    $sas = $testdrop.accessData.sasKey
+
+    $azcopy = Get-ToolPath -Name "AzCopy\AzCopy.exe"
+    Write-Verbose "Calling AzCopy = $azcopy" -Verbose
+
+    $azlog = ("{0}\..\azlog" -f $src)
+    $args = ("/Source:`"{0}`" /Dest:{1} /DestSAS:{2} /S /Z:`"{3}`"" -f $src, $dest, $sas, $azlog)
+
+    # Create a file with the arguments as the content
+    $responseFile = [System.IO.Path]::GetTempFileName()
+    [System.IO.File]::WriteAllText($responseFile, $args)
+
+    $responseFileArgs = [String]::Format("/@:""{0}""", $responseFile)
+    try {
+        Write-Verbose "Calling AzCopy tool with file $responseFile" -Verbose 
```

Comment:
Do we want to put the file name in verbose log.. People can easily look into the file and get the sas key..